docs/src/references/c8y-log-management.md

```diff
@@ -0,0 +1,122 @@
+# Log file management from Cumulocity
+
+Thin-edge provides an operation plugin to [fetch log files on the device from Cumulocity](https://cumulocity.com/guides/users-guide/device-management/#logs).
+
+* Log file management from Cumulocity is provided with a `c8y_log_plugin` which runs as a daemon on thin-edge.
+* The device owner can define the list of log files that can be retrieved from Cumulocity,
+  in the plugin's configuration file named `c8y_log_plugin.toml`.
```

Comment:
I would remember the definition of a log type. This term is used above without any explanation. It might be clear for most Cumulocity users but certainly not for all. 

```
* Each record of the plugin's configuration file defines a type of log that is a logical group of log files.
```